### PR TITLE
Update German Stringtable

### PR DESCRIPTION
**Pull request by licht-im-Norden87**
excluding a lot of lines in the medical because iam not familiar with these therms.

### CODE REVIEW

File: addons/interaction/stringtable.xml [hunk 1]
```diff
@@ -654,7 +662,7 @@
       <Spanish>Interactuar</Spanish>
     </Key>
     <Key ID="STR_ACE_Interaction_Passengers">
-      <English>Passengers</English>
+      <English>Fahrzeuginsassen</English>
```
Comment (esteldunedain) [inline addons/interaction/stringtable.xml]: Oops, this is wrong.

**Comment by licht-im-Norden87** [comment]
Corrected ;)

**Comment by licht-im-Norden87** [comment]
I'd suggest to exclude my medical translation since "Herr Stein" translated a lot more in this specific stringtable.

**Comment by KoffeinFlummi** [comment]
Thanks! Please add your email address to your entry in the AUTHORS file before we merge this.

**Pull request merged**